### PR TITLE
Image Upload With cloudinary and user banner

### PR DESCRIPTION
**Pull request by siyamhosan**
## Fixes Issue
Image upload from base64 encoded to using cloudinary

## Proposed Changes
- Faster profile load.
- User Banner added

## Screenshots
image

## Reason
So I was here to add a user banner to the profile page, I successfully did that then I saw the avatar and banner was stored to database as base64 string. by this page loading was taking a long time becuase prisma was struggling to get this much data from db.
So I used Cloudinary for image upload handling and used there image link in src.

## Implementation
 if this good enough, you need to create a Cloudinary account then get the cloud name and put it in env
then you need to go to `settings > upload > scroll to upload presets`

create 2 upload presets 
- for avatar
- for banner 

then use these upload presets in `\pages\admin\settings\index.jsx`:167 and 212

### CODE REVIEW

**Comment by urdadx** [comment]
Hey, @siyamhosan, thank you for sending this pull request. It is a great improvement to Librelinks.
However, I don't think adding a banner to the user's page is necessary. Although great, it makes the page look a little cluttered. I want to keep things as simple as possible.

If you can, please open a new PR without adding the banner.

File: prisma/schema.prisma [hunk 1]
```diff
@@ -1,39 +1,37 @@
...
-  totalViews     Int       @default(0)
-  createdAt      DateTime  @default(now())
-  updatedAt      DateTime  @updatedAt
-  links          Link[]
+  id            String    @id @default(auto()) @map("_id") @db.ObjectId
+  name          String?
+  handle        String?   @unique
+  bio           String?
+  image         String?
+  banner        String?
```
Comment (urdadx) [inline prisma/schema.prisma:17]: Adding a banner to the user's page goes against my initial plan of making the site minimalistic

**Review by urdadx — CHANGES_REQUESTED** [review]
Send a new PR without the banner image

File: prisma/schema.prisma (hunk 1 above)
Comment (PriyansuMaurya) [reply prisma/schema.prisma:17]: Yeah! I agree.

**Pull request closed without merging**